edsdme/blocks/announcements-preview/announcements-preview.css

```diff
@@ -0,0 +1,74 @@
+/* stylelint-disable */
+.announcement-item,
+.link-wrapper {
+  display: flex;
+  align-items: center;
+  margin: 26px 0px;
+  background-color: white;
+  width: 100%;
+  height: 130px;
+  max-width: 600px;
+}
+
+announcements-preview .con-button {
+  margin: 5px 0px;
+}
+
+.card-image {
+  min-width: 100px;
+  width: 20%;
+  height: 100%;
+  margin-right: 20px;
+  overflow: hidden;
+}
+
+.card-image picture,
+.card-image img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.card-content {
+  height: 100%;
```

Comment:
there's no margin/padding to the right of the card content, which means that the title and description can bump up to the edge of the card. Looks a little odd.

<img width="615" alt="Screenshot 2024-10-16 at 10 36 59" src="https://github.com/user-attachments/assets/7b22b1b1-67e8-48db-83cd-2d6aa50eefc3">